variables.tf

```diff
@@ -21,6 +21,23 @@ variable "name" {
   nullable    = false
 }
 
+variable "bypass_platform_safety_checks_on_user_schedule_enabled" {
+  type        = bool
+  description = "(Optional) Specifies whether to skip platform scheduled patching when a user schedule is associated with the VM. Only valid if patch_mode is `AutomaticByPlatform`."
+  nullable    = false
+  default     = false
+}
+
+variable "reboot_setting" {
+  type        = string
+  description = "(Optional) Specifies the reboot setting for platform scheduled patching. Possible values are `Always`, `IfRequired` and `Never`. Only valid if `patch_mode` is `AutomaticByPlatform`."
+  default     = null
+  validation {
+    condition     = reboot_setting == null ? true : contains(["Always", "IfRequired", "Never"], var.reboot_setting)
```

Comment:
@davidkarlsen We still have a syntax error here, it should be `var.reboot_setting == null` instead of `reboot_setting == null`.